data/transition-sites/gds_alphagov_licensify_admin.yml

```diff
@@ -0,0 +1,8 @@
+---
+site: gds_alphagov_licensify_admin
```

Comment:
Could you hyphenate the app name in `site` and the filename as we did for the other apps, ie `gds_alphagov_licensify-admin`?

Otherwise :+1: